### PR TITLE
Some golint fixes

### PR DESCRIPTION
**Pull request by mbhinder**
Noticed some golint errors, cleaned them up. There are more, can try to go through them as well.

### CODE REVIEW

File: defaults.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func (dns *Msg) SetIxfr(z string, serial uint32, ns, mbox string) *Msg {
 	dns.Question = make([]Question, 1)
 	dns.Ns = make([]RR, 1)
 	s := new(SOA)
-	s.Hdr = RR_Header{z, TypeSOA, ClassINET, defaultTtl, 0}
+	s.Hdr = RR_Header{z, TypeSOA, ClassINET, defaultTTL, 0}
```
Comment (miekg) [inline defaults.go]: The Ttl -> TTL ship has sailed. Go DNS uses Ttl (although TTL would have been better in hindsight)

File: ztypes.go [hunk 2]
```diff
@@ -282,16 +282,16 @@ func (rr *DNAME) len() int {
 func (rr *DNSKEY) len() int {
 	l := rr.Hdr.len()
 	l += 2 // Flags
-	l += 1 // Protocol
-	l += 1 // Algorithm
+	l++    // Protocol
```
Comment (miekg) [inline ztypes.go]: I like the old style because it's inline with `l +=2` directly above it

File: msg.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ var (
 	ErrTruncated     error = &Error{err: "failed to unpack truncated message"} // ErrTruncated indicates that we failed to unpack a truncated message. We unpacked as much as we had so Msg can still be used, if desired.
 )
 
-// Id, by default, returns a 16 bits random number to be used as a
+// Id by default, returns a 16 bits random number to be used as a
```
Comment (miekg) [inline msg.go]: The doc changes are still welcome :) Although I nacked the rest of your changes. Are you willing to make that change?

Comment (mbhinder) [inline msg.go]: Done, thanks for reviewing

**Comment by miekg** [comment]
Thanks. Doc fixes are always useful

**Pull request merged**